connector/mqtt/vernemq/v2k-brigde/README.md

```diff
@@ -37,3 +34,5 @@ V2K_MQTT_SUBSCRIPTION_QOS| Mqtt client Quality of service
 V2K_MQTT_CA_FILE         | Mqtt client ca file                                                 | ${BASE_DIR}/app/verne/${HOSTNAME}.ca | string    |
 V2K_MQTT_CERT_FILE       | Mqtt client certificate file                                        | ${BASE_DIR}/app/verne/${HOSTNAME}.crt| string    |
 V2K_MQTT_KEY_FILE        | Mqtt client key file                                                | ${BASE_DIR}/app/verne/${HOSTNAME}.key| string    |
+V2K_BACKPRESSURE_MAX_QUEUE_LENGTH | Mqtt Client backpressure messages length in bytes          | 1048576                              | integer   |
```

Comment:
Change from `Mqtt Client backpressure messages length in bytes` to `Maximum backpressure queue length in bytes`